package.json

```diff
@@ -96,6 +96,7 @@
   },
   "devDependencies": {
     "@angular-devkit/build-angular": "^0.10.0",
+    "@angular-devkit/schematics": "^7.3.3",
```

Comment:
Remove all changes to the package.json and yarn.lock files.

Reply:
Is this the reason build is getting failed?

Reply:
Rolled back.